package.json

```diff
@@ -23,6 +23,11 @@
     "standard"
   ],
   "author": "Ben Coe <ben@npmjs.com>",
+  "contributors": [
+    {
+      "name": "Andrej Zachar <andrej@chocolatejar.eu>"
```

Comment:
I don't usually keep this updated; instead letting people click the contributors link in GitHub; if you'd like to toss your name here, mind also adding all the other folks listed under contributors; alternatively let's take this out 👍